packages/node_modules/@webex/plugin-meetings/test/unit/spec/member/index.js

```diff
@@ -0,0 +1,25 @@
+import sinon from 'sinon';
+import {assert} from '@webex/test-helper-chai';
+
+import MemberUtil from '@webex/plugin-meetings/src/member/util';
+import Member from '@webex/plugin-meetings/src/member';
+
+
+describe('member', () => {
+  const sandbox = sinon.createSandbox();
```

Comment:
nitpick: According to the [sinon docs](https://sinonjs.org/releases/v9/sandbox/#default-sandbox) you can probably avoid doing this (since there isn't a specialised setup), and instead just use the `sinon` import wherever you had `sandbox` before. Doesn't make any functional difference though, just avoids unnecessary boilerplate.